mon-pix/app/routes/assessments.js

```diff
@@ -1,13 +1,16 @@
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 
 export default class AssessmentsRoute extends Route {
+  @service intl;
+
   model(params) {
     return this.store.findRecord('assessment', params.assessment_id);
   }
 
   afterModel(model) {
     if (model.isCertification) {
-      model.title = `Certification ${model.title}`;
+      model.title = this.intl.t('page-title.assessments-certification') + model.title;
```

Comment:
```suggestion
      model.title = this.intl.t('page-title.assessments-certification', {certificationNumber: model.title});
```
Avec 

`    "assessments-certification": "Certification {certificationNumber}",`